app/app/controllers/webhooks/pinwheel/events_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Webhooks::Pinwheel::EventsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def create
```

Comment:
In general this method seems to be doing a lot. It violates the single responsibility principal. It's not only handling the creation of two different webhooks, but first handles validating a request signature. This means in order to test that the webhooks are processed correctly we need mock a valid request signature.

I would suggest that the `create` method handle delegating the processing of `paystubs.added` and `accounts.added`. 

```suggestion
def create
  switch params["event"]
  when "paystubs.added"
    handle_paystub_added(params)
  when "accounts.added"
    handle_accounts_added(params)
  end
end
```

Additionally, we can move the logic to validate a request to its own method.